config/initializers/devise.rb

```diff
@@ -370,7 +370,11 @@
   # When using the :trackable module, set to true to consider magic link tokens
   # generated before the user's current sign in time to be expired. In other words,
   # each time you sign in, all existing magic links will be considered invalid.
-  # config.passwordless_expire_old_tokens_on_sign_in = true
+  config.passwordless_expire_old_tokens_on_sign_in = true
```

Comment:
```suggestion
  # config.passwordless_expire_old_tokens_on_sign_in = true
```

This should remain commented from https://github.com/DFE-Digital/apply-for-qualified-teacher-status/pull/1505.